tests/ci.benchmarks/.gitignore

```diff
@@ -4,3 +4,4 @@
 datasets/*.rdb
 *.rdb
 binaries
+profile_*
```

Comment:
In which directory those file actually created in?

Reply:
> In which directory those file actually created in?
```
tests/ci.benchmarks/profile_*
```
Will adjust it :)

Reply:
already saw that it's in this folder gitignore. I believe this is the correct way to do it (meaning no change). Agree?

Reply:
yep.